manifests/08-prometheus_rule.yaml

```diff
@@ -29,4 +29,13 @@ spec:
       for: 5m
       labels:
         severity: info
+    - alert: InsightsRecommendationActive
+      annotations:
+        summary: An Insights recommendation is active for this cluster.
+        description: Rule ID {{ $labels.rule_id }}, Error Key {{ $labels.error_key }}, Total Risk {{ $labels.total_risk }}, Info Link {{ $labels.info_link }} Description {{ $labels.description }}
```

Comment:
nit: I would move the info link more to the beginning. I guess nobody cares about error key and rule id..... :) WDYT @tdosek ?

Reply:
The current format of the description is purely for debugging purposes, so that I can easily see what the individual properties are.